sourcegraph/about
# Update 2 hosted PDFs

Closes #5666. Updates 2 hosted PDFs, compresses files, and reinstates meta titles.

### Changelog
""

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure no breaking changes & updates seen on `/guides/transform-your-codebase-into-knowledge-actions-and-insights.pdf` and `/guides/data-driven-relationships-with-codebases.pdf`
